packages/ui/src/prototypes/typography.stories.jsx

```
@@ -12,22 +12,20 @@ export default {
 };
 
 export const Typography = () => (
- <main className={css.main}>
-   <Container dangerouslySetInnerHTML={{ __html: content }} />
- </main>
+  <main className={css.main}>
+    <Container dangerouslySetInnerHTML={{ __html: content }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)